src/Entity/Entity.php

```diff
@@ -157,6 +157,22 @@ public function getCreator()
         return $this->creator;
     }
 
+    /**
+     * Getter for creator full name.
+     *
+     * @access public
+     *
+     * @return string.
+     */
+    public function getCreatorName(): string
+    {
+        $creatorName = '';
+        if ($this->getCreator()) {
+            $creatorName = $this->getCreator()->getFirstName() . ' ' . $this->getCreator()->getLastName();
+        }
+        return $creatorName;
```

Comment:
```suggestion
        return $creatorName ?? '';
```

Reply:
I think initializing the variable to a string is better so that we dont get the variable not defined warning